run_dir/static/js/project_samples.js

```diff
@@ -1013,9 +1013,10 @@ function load_charon_summary(){
       if (data['tot'] != 0){
           $('#charon-status').show();
           $('#charon-status-tot').text(data['tot']);
+          $('#charon-status-ab').text(data['ab']);
```

Comment:
It's coherent with what charon outputs : http://charon.scilifelab.se/api/v1/summary
The template uses full names. I shorten them because I have aggregate classes like passed_unaborted, and I might have more. 
hge is Human genome equivalent, but you can find it in the template.